docs/pages/setup/security/reduce-blast-radius.mdx

```diff
@@ -0,0 +1,134 @@
+---
+title: Reducing the Blast Radius of Attacks
+description: You can easily configure Teleport to minimize the scope of security breaches
+---
+
+Teleport encourages users to practice defense in depth so that every component of their infrastructure is protected against an attack, even if the attack is partially successful. You can configure Teleport to add layers of protection to your cluster when users authenticate or request elevated privileges. In this guide, we will show you how to:
+
+- [Make MFA mandatory for `tsh` logins](#make-mfa-mandatory-for-tsh-login)
+- [Present an MFA challenge for every attempt to access a resource](#present-an-mfa-challenge-for-every-attempt-to-access-a-resource)
+- [Require multiple reviews for role requests](#require-multiple-reviews-for-role-requests)
+- [Automatically prevent some roles from requesting others](#automatically-prevent-some-roles-from-requesting-others)
+
+
+## Make MFA mandatory for `tsh login`
+If a user sets up an account to authenticate to their Teleport cluster with only a password, an adversary can [gain access](/blog/no-passwords) to the password using brute-force attacks, person-in-the-middle attacks, or phishing. But even if a user's password is compromised, you can stop an attacker from authenticating with it when they run `tsh login`. 
```

Comment:
nit: i'd remove this link to keep the readers flow. Take a look at some tips here:

https://www.divio.com/

and

https://goteleport.com/docs/docs/

We can add useful links in "Next steps" last section

Reply:
I'll move most inline links to a "next steps" section.